src/DIRAC/Resources/Computing/CREAMComputingElement.py

```diff
@@ -349,9 +349,28 @@ def __parseJobStatus(self, output):
 
         return resultDict
 
-    def getJobOutput(self, jobID, localDir=None):
-        """Get the specified job standard output and error files. If the localDir is provided,
-        the output is returned as file in this directory. Otherwise, the output is returned
+    def getJobLog(self, jobID):
+        """Get job logging info
+
+        :param str jobID: job identifier
```

Comment:
Here, "job" is effectively the pilot for DIRAC. Please just remind this in this docstring.